lib/public_key/src/pubkey_os_cacerts.erl

```diff
@@ -169,8 +169,11 @@ bsd_paths() ->
      "/etc/openssl/certs/ca-certificates.crt"
     ].
 
-sunos_path() ->
-    "/etc/certs/CA/".
+sunos_paths() ->
+    ["/etc/certs/CA/", %% Oracle Solaris, some older illumos distros
+     "/etc/ssl/cacert.pem", %% OmniOS
+     "/opt/local/etc/openssl/certs/ca-certificates.crt" %% SmartOS and pkgsrc
```

Comment:
Is there no link on this from `/etc/openssl` as in the `bsds`?
I would prefer to only have `/etc/` stuff there?